vars/scanRpms.groovy

```diff
@@ -65,7 +65,7 @@ def call(Map config = [:]) {
 
   provisionNodes NODELIST: nodelist,
                  node_count: 1,
-                 distro: stage_info['ci_target'],
+                 distro: (stage_info['ci_target'] =~ /([a-z]+)(.*)/)[0][1] + stage_info['distro_version'],
```

Comment:
This disables using using a pragma or command line parameter to specify an test CI image.

Reply:
Probably.  All of the target handling needs to be refactored to deal with current requirements.  Mainly this is just more fallout from having to be able to support multiple (EOL, unsupported even) point releases.